demo/www/js/index2.js

```diff
@@ -1,3 +1,7 @@
+/* global txtArea:true*/
+/*global btnDismiss:true*/
+/*global PGMultiView:true*/
+/*global data:true*/
```

Comment:
`data` actually is undefined, you should fix this one by adding the `var data = txtArea.value;` or get rid of it and just call `PGMultiView.dismissView(txtArea.value);`